### DISCUSSION TITLE
A mechanism to report obviously malicious users is needed

### DISCUSSION THREAD

**Discussion by alexpyattaev** [rust-lang/crates.io#7081] [General]
Recently stumbled upon this user:
https://crates.io/users/r3v2d0g?page=8&sort=downloads

The guy has 407 crates published, with pretty much all of them having no content of any sort. While none of this is strictly malicious activity as such, claiming 407 crate names and using at best 3 of them is, IMHO, quite excessive, and would make publishing new crates with sensible names harder and harder over time. It would be nice if there existed a mechanism to "report" users for borderline behavior such that damage they cause can be mitigated.

**Comment by Turbo87** [comment]
unfortunately, this is currently not forbidden by the crates.io policies, unless anyone can prove that these crates were published in some form of automated way. we are trying to update our policies with https://github.com/rust-lang/rfcs/pull/3463 to explicitly disallow such behavior. feel free to chime in over there.

**Reply by alexpyattaev** [reply]
Well, if not automated, the person doing was sure very persistent! I mean, there is over 400 of them...

**Reply by Turbo87** [reply]
- the RFC has been accepted at the end of 2023
- the spam of https://crates.io/users/r3v2d0g has been deleted
- you can report similar issues at https://crates.io/support or with the "Report crate" button on each crate page

I consider this resolved now :)

**Reply by alexpyattaev** [reply]
Thank you!
